UMI3D-Browser-Desktop/Assets/Dependencies/UMI3D SDK/ClientDevelopmentKit/Collaboration/Runtime/Networking/UMI3DWorldControllerClient.cs

```diff
@@ -126,6 +135,22 @@ private async Task<bool> Connect(ConnectionDto dto)
                     };
                     return await Connect(_answer);
                 }
+                else if (answerDto is LibrariesToDownloadDto librariesToDownload)
```

Comment:
You should have a dto with ConnectionFormDto and LibrariesToDownloadDto in it.
So only one if for those two cases.